src/shared/dropdown-list-v2/dropdown-list.styles.tsx

```diff
@@ -188,3 +196,12 @@ export const Spinner = styled(ComponentLoadingSpinner)`
     margin-right: ${Spacing["spacing-8"]};
     color: ${Colour["icon"]};
 `;
+
+export const NoResultDescContainer = styled.div<ContainerStyleProps>`
+    padding: 0 ${Spacing["spacing-16"]} ${Spacing["spacing-12"]}
+        ${Spacing["spacing-16"]};
+    ${(props) =>
+        props.$variant === "small"
+            ? Font["body-sm-regular"]
+            : Font["body-md-regular"]}
```

Comment:
from DS Figma it will apparently inherit the existing font size, so this custom font style is not needed